src/app/components/sites/site/site.component.ts

```diff
@@ -93,18 +90,23 @@ class SiteComponent extends PageComponent implements OnInit {
   }
 
   private getOldestDates() {
-    this.audioRecordingsApi
-      .filter({
-        filter: { ["sites.id"]: { eq: this.site.id } } as any,
-        sorting: { orderBy: "recordedDate", direction: "asc" },
-        paging: { items: 1 },
-      })
+    this.filterByDates("asc", { paging: { items: 1 } })
       .pipe(takeUntil(this.unsubscribe))
       .subscribe(
         (recordings) => (this.recordingsStart = recordings[0]?.recordedDate),
         (err) => console.log({ err })
       );
   }
+
+  private filterByDates(
+    direction: "desc" | "asc",
```

Comment:
this "desc" | "asc" string literal type is hardly unique. It should probably be referring to an existing type/alias?